apps/pie-docs/src/_11ty/shortcodes/componentDetailsTable.js

```diff
@@ -5,6 +5,10 @@
  * @returns {string} - The HTML representation of a table row.
  */
 const buildRow = (cells) => cells.map((cell) => {
+    if (cell === null) {
+        return '<td></td>';
```

Comment:
This means we can have an empty cell if we want, like for a string field it doesn't make sense (to me) to say what the "options" are.

Reply:
Compare [this props table](https://pr1002-docs.pie.design/components/card/code/#props) with the bottom half of [button's props table](https://pie.design/components/button/code/#props)

Reply:
<img width="1233" alt="Screenshot 2023-11-21 at 12 23 34" src="https://github.com/justeattakeaway/pie/assets/16766185/7cca54b8-6c61-4676-9864-f2476e8e32be">

Not sure if related but I'm not sure what I would do with this `aria` prop as a consumer

Reply:
It's a fair point, I think at the moment it only has an optional label property. I'll give it a go